packages/features/bookings/Booker/components/BookEventForm/BookEventForm.tsx

```diff
@@ -115,13 +115,13 @@ export const BookEventForm = ({
         {!isPlatform && IS_CALCOM && (
           <div className="text-subtle my-3 w-full text-xs opacity-80">
             <Trans i18nKey="signing_up_terms">
-              By proceeding, you agree to our{" "}
+              {t("agree_to_our")}{" "}
```

Comment:
This is not how you should use i18n here. check Trans https://react.i18next.com/latest/trans-component